qiskit_nature/components/variational_forms/adapt_ucc.py

```diff
@@ -0,0 +1,34 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+TODO.
+"""
+
+import logging
+
+from .adaptive_ansatz import AdaptiveAnsatz
+from .ucc import UCC
+
+logger = logging.getLogger(__name__)
+
+
+class AdaptUCC(UCC, AdaptiveAnsatz):
```

Comment:
If these are all going to be Blueprint circuits then the should so to `circuit` and some sub-folder there `ansatzes` ? (The base class will evidently migrate over to Terra)

Reply:
Yes I agree. This was also already mentioned above by @Cryoris